docs/preview/02-Features/powershell/azure-integration-account.md

```diff
@@ -271,4 +271,80 @@ Uploading a *single private certificate* into an Integration Account and set add
+# Uploading partner 'MyPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'MyPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+```
+
+Uploading a *single partner* into an Integration Account and set add a prefix to the name of the partner within the Integration Account.  
+```powershell
+PS> Set-AzIntegrationAccountPartners -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -PartnerFilePath "C:\Partners\MyPartner.json" -ArtifactsPrefix 'dev-'
+# Uploading partner 'dev-MyPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'dev-MyPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+```
+
+Uploading *all partners* located in a specific folder into an Integration Account.  
+```powershell
+PS> Set-AzIntegrationAccountPartners -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -PartnersFolder "C:\Partners"
+# Uploading partner 'MyFirstPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'MyFirstPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+# Uploading partner 'MySecondPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'MySecondPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+```
+
+Uploading *all partners* located in a specific folder into an Integration Account and set add a prefix to the name of the partners.
+```powershell
+PS> Set-AzIntegrationAccountPartners -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -PartnersFolder "C:\Partners" -ArtifactsPrefix 'dev-'
+# Uploading partner 'dev-MyFirstPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'dev-MyFirstPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+# Uploading partner 'dev-MySecondPartner.json' into the Azure Integration Account 'my-integration-account'
+# Partner 'dev-MySecondPartner.json' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+```
+
+**Partner JSON Example**
+The partner definition is the JSON representation of your partner, this JSON definition can also be viewed in the Azure Portal using https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners#edit-a-partner and clicking on `Edit as JSON`.
+An example of this file:
+```
+{
+  "name": "MyPartner",
+  "properties": {
+    "partnerType": "B2B",
+    "content": {
+      "b2b": {
+        "businessIdentities": [
+          {
+            "qualifier": "1",
+            "value": "12345"
+          },
+          {
+            "qualifier": "1",
+            "value": "54321"
+          }
+        ]
+      }
+    }
+  }
+}
```

Comment:
Great addition to this! 👍 Will definitely come in handy when people read the docs. Thanks!